modules/@apostrophecms/login/ui/apos/components/TheAposLogin.vue

```diff
@@ -102,7 +102,10 @@ export default {
       try {
         await apos.http.post(`${apos.modules['@apostrophecms/login'].action}/login`, {
           busy: true,
-          body: this.doc.data
+          body: {
+            ...this.doc.data,
+            session: true
```

Comment:
Not sure I understood why this property has been added (still able to log in through the UI)

Reply:
Without this property you get a bearer token instead, which we anticipate will be the more "normal" case for everyone else but us (third party apps logging in).